stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -3152,7 +3152,8 @@ internal struct _Native${Self}Buffer<${TypeParameters}> {
 extension _Native${Self}Buffer 
   where ${'Key' if Self == 'Dictionary' else 'Element'} : Hashable
 {
-  internal typealias HashTypedStorage = _Native${Self}Storage<${TypeParameters}>
+  internal typealias HashTypedStorage = 
```

Comment:
It could be, but honestly it shows up in like one or two places so it might as well be a bit verbose for clarity. It's mostly here to put all the GYB gunk in one place.